app/controllers/api/v1/medias_controller.rb

```diff
@@ -1,42 +1,53 @@
 require "csv"
 module Api
-	module V1
-		class MediasController < ApplicationController
+  module V1
+    class MediasController < ApplicationController
 
-# Read file path from API, generate asset ID and Pass the job to AddMetadataJob
-#Parameter: Path to CSV file
-		
-		def create
-			@asset_id=('0'..'z').to_a.shuffle.first(8).join
-			@path= params[:csv_location]
-			@a_id= params[:account_id]
-			@data=CSV.read(@path)
-			#MetadataWorker.perform_async(@data,@a_id)
-			AddMetadataJob.perform_later(@path,@a_id,@asset_id)
-		end
+#Read file path from API, generate asset ID and Pass the job to AddMetadataJob
+#Input: Path to CSV file
+	  def create
+		@path = params[:csv_location]
+		@account_id = params[:account_id]
+		@data = CSV.read(@path)
+		#MetadataWorker.perform_async(@path,@account_id,@asset_id)
+		AddMetadataJob.perform_later(@path, @account_id)
+	  end
 
-# Get the account ID from the request and list all the medias persent in that account
-		def index
-			@medias = Media.where(account_id: params[:account_id])
+#Get the account ID from the request and list all the medias persent in that account
+	  def index
+	    if params[:asset_id].present?
```

Comment:
how to handle multiple params together? i need to apply title and max_duration filter

Reply:
should it give 2 results or one result with both the filter applied?

Reply:
both filter has to be applied together

Reply:
okay